dev_docs/contributing/third_party_dependencies.mdx

```diff
@@ -67,6 +67,57 @@ on Github:
   [Permitted Open Source Licenses list](https://github.com/elastic/open-source/blob/main/elastic-product-policy.md#permitted-licenses-list).
 </DocCallOut>
 
+### Dependency evaluation
```

Comment:
For this stopgap period, we should explain when to follow this procedure. I think we should follow this for all new dependencies, but we shouldn't expend effort reviewing dependency upgrades at this time. Reviewing upgrades would be too time consuming, given that we have Renovate opening PRs on a daily basis.

cc @elastic/kibana-security for discussion.

Reply:
Agreed + moving deps from dev to production (likely very-very rare case) 👍